entity-framework/core/providers/sqlite/limitations.md

```diff
@@ -70,7 +70,7 @@ Delete               | ✔
 
 ### Migrations limitations workaround
 
-You can workaround some of these limitations by manually writing code in your migrations to perform a rebuild. Table rebuilds involve creating a new table, copying data to the new table, dropping the old table, renaming the new table. You will need to use the `Sql(string)` method to perform some of these steps.
+You can workaround some of these limitations by manually writing code in your migrations to perform a rebuild. Table rebuilds involve creating a new table, copying data to the new table, dropping the old table, renaming the new table. You will need to use the [`MigrationBuilder.Sql`](/dotnet/api/microsoft.entityframeworkcore.migrations.migrationbuilder.sql) method to perform some of these steps.
```

Comment:
Use `<xref>` to refer to our API documentation (and to other pages in the conceptual docs).